docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+services:
+  mosquitto-mqtt-server:
+    image: "eclipse-mosquitto"
+    ports:
+        - "1883:1883"
```

Comment:
Is this format (`1883:1883`) a convention or does the two numbers represent different stuff?